proto/node/akash/deployment/v1beta3/deploymentmsg.proto

```diff
@@ -44,7 +44,8 @@ message MsgCreateDeployment {
 // MsgCreateDeploymentResponse defines the Msg/CreateDeployment response type.
 message MsgCreateDeploymentResponse {}
 
-// MsgDepositDeployment deposits more funds into the deposit account
+// MsgDepositDeployment deposits more funds into the deposit account.
+// Deprecated: deployment deposits in future versions are part of the escrow module.
```

Comment:
disregard all changes to deployment/v1beta3, as codegeneration breaks upgrade migrations

Reply:
just `deployment/v1beta3`. do not edit comment as well, it triggers code generation